packages/deployment-service/src/functions/create.pipeline.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { createPipeline } from './create.pipeline'
```

Comment:
It's a small point but can tests live in a `__tests__` folder please? It's just the convention in the repo